addons/web_settings_dashboard/static/src/js/dashboard.js

```diff
@@ -346,10 +346,22 @@ var DashboardTranslations = Widget.extend({
     template: 'DashboardTranslations',
 
     events: {
-        'click .o_load_translations': 'on_load_translations'
+        'click .o_load_translations': '_onLoadTranslations'
```

Comment:
commit message: 'it redirects to 1st app (community) or *missing something?* (enterprise)'